tools/github-team-user-store/README.md

```diff
@@ -0,0 +1,28 @@
+# GitHub Team User Store
+
+## Overview
+
+The github-team-user-store is an internal only tool. The tool will get the list of teams/users under azure-sdk-write and store the results as a json string in Azure blob storage. The purpose of this is to enable team usage in CODEOWNERS by allowing the CodeOwnersParser to pull the list of users when it encounters a non-user, or team, entry. The reason why this code isn't directly in the CodeOwnersParser is because it requires a specific, fine-grained Personal Access Token (PAT) in order to be able to get the users for teams which isn't something that's readily available or prudent to be added to all of the places where CodeOwnersParser is used.
```

Comment:
Can we clarify what kind of entity here is the `azure-sdk-write` ? Eg. env variable.

Reply:
azure-sdk-write is a team. It's the team that everything with write permissions is supposed to inherit from. Because these teams are going to be in CODEOWNERS they're all required to have write permission. Rather than getting all 1933 teams in the Azure organization, it was decided that, because of the write permission limitation, to only get the teams under azure-sdk-write. 

I'll clarify the comment with the next commit.